UTC+01/FR/PAC/FR-PAC-Hautes-Alpes/settings.sh

```diff
@@ -39,7 +39,7 @@ PTNA_WWW_REGION_LINK="https://overpass-turbo.eu/map.html?Q=[out%3Ajson][timeout%
 
 # Name + Link to the network provider / transport association
 PTNA_WWW_NETWORK_NAME="L'Agglo en bus;Bustub Briançon;Navettes du Queyras"
```

Comment:
Forget to change the name of the operator as well instead of 'Bustub Briançon' this should be 'AltiGo'